pymetis/__init__.py

```diff
@@ -257,20 +263,47 @@ def part_mesh(n_parts, connectivity, options=None):
 
     n_elements = len(connectivity)
     n_vertex = len(set(conn))
-
+    
+   
     # Handle option validation
     if options is None:
         options = Options()
+        #set default values
+        options.ptype = metis.PType.KWAY
+        options.objtype = metis.ObjType.CUT
+        options.ctype = metis.CType.SHEM
+        options.dbglvl = 0
+        options.minconn = 0
+        options.contig = 0
+        options.niter = 10
+        options.ncuts = 1
 
     if options.numbering not in [-1, 0]:
         raise ValueError("METIS numbering option must be set to 0 or the default")
 
+    if tpwgts is None:
+        tpwgts = []
+    if len(tpwgts) > 0:
+        if len(tpwgts) != n_parts:
+            raise RuntimeError("The length of tpwgts mismatches `n_part`")
+
+        import numpy as np
```

Comment:
pymetis has no declared dependency on numpy, and I am not looking to change that.

Reply:
I saw `requirements.txt` mentioned numpy so I used it. I will remove it in the next commit.

Reply:
That's what's needed for tests. `setup.py` declares dependencies.